### DISCUSSION TITLE
Error node version

### DISCUSSION THREAD

**Discussion by galaksy2020** [Koenkk/zigbee2mqtt#24176] [General]
Hi everybody,
I just re-installed zigbee2mqtt on unraid with the latest version and i've got the following warning message in logs :
" Zigbee2MQTT requires node version ^18 || ^20 || ^22, you are running v21.7.3!"
Strange because I had no issues with the previous installation ...
Do you know how to fix it please ?
Thanks in advanced :)
Marc

**Comment by lux73** [comment]
Install nodejs 22 - works without Problems for me

your v21.x is not supported anymore

**Reply by galaksy2020** [reply]
ok thanks, I came to the same conclusion ;) I gonna do it this afternoon.
But it's quite strange because my unraid and zigbee2mqtt version didn't change since my previous installation without issues last week ...
I was wondering if there was not some others issues.
